scarb/src/compiler/compilers/starknet_contract.rs

```diff
@@ -78,6 +78,15 @@ impl ContractSelector {
         PackageName::new(parts.0)
     }
 
+    fn is_wildcard(&self) -> bool {
+        self.0.ends_with("::*")
```

Comment:
Please refer to `::` as `CAIRO_PATH_SEPARATOR` wherever possible.

Reply:
implemented GLOB_PATH_SELECTOR const

Reply:
added GLOB_PATH_SEPARATOR const